packages/core/src/ceramic.ts

```diff
@@ -617,6 +617,7 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
       if (this._metricsConfig?.metricsPublisherEnabled) {
         // First, subscribe the node to the Model used for NodeMetrics
         const metricsModel = NodeMetrics.getModel(this._networkOptions.name)
+        await this._waitForMetricsModel(metricsModel)
```

Comment:
make sure this has a timeout in case it doesn't work?  let just put some tight error conditions around it i think